src/ca65/main.c

```diff
@@ -307,6 +307,14 @@ static void SetSys (const char* Sys)
             NewSymbol ("__OSIC1P__", 1);
             break;
 
+        case TGT_CREATIX_EMUL:
+            NewSymbol ("__CREATIX_EMUL__", 1);
+            break;
+            
+        case TGT_CREATIX_NATIVE:
```

Comment:
The emulation layer is needed by programs that are written in C (the compiler doesn't generate the C39 addressing modes).  Assembly-only programs might not need that layer; therefore, it wouldn't need to be installed in the PROM.

But, you do have a valid point.  Maybe, that issue could be handled by an ld65 command-line option.  The `-u` option could be used to force the layer's code to be linked into a ROM program.  (Or, two configuration files could be created; that choice was done for some other targets, such as the C64.)  Only one library would need to be built.